runtime/testy/runtime.py

```diff
@@ -218,6 +228,11 @@ def startHibike(badThingsQueue, stateQueue, pipe):
   except Exception as e:
     badThingsQueue.put(BadThing(sys.exc_info(), str(e)))
 
-if __name__ == "__main__":
-  runtimeTest()
+parser = argparse.ArgumentParser()
```

Comment:
You'll want to put all this code inside an if __name__ == "__main__" block
That ensures that this code will only be run if this is the main file being executed, and that if you import runtime.py, argument parsing will not run.